index.js

```diff
@@ -3,7 +3,12 @@ import { listenKeys } from 'nanostores'
 
 export function useStore(store, opts = {}) {
   let [, forceRender] = useState({})
+  let [valueBeforeEffect] = useState(store.get())
```

Comment:
Why do we need to wrap it to `useState`? Why not just `valueBeforeEffect = store.get()`?

Reply:
If you use `valueBeforeEffect = store.get()`, then `valueBeforeEffect` will change along with `store.get()` and so `useEffect` will not run `forceRender({})`

When I wrap `let [valueBeforeRender] = useState(store.get())`, the first value of the store will be settled in `valueBeforeRender` only one times. And `valueBeforeRender` will not change, if store be updated